### ISSUE TITLE
[a11y] Product page - Drop elements not accessible

### ISSUE THREAD

**Issue by g123k** [openfoodfacts/openfoodfacts-server#10293]
Hi everyone,

To improve accessibility on the website, we have some "simple" fixes to implement.
On the product page, we have several `drop_xxx` elements, that need to be fixed:

> aria-describedby and aria-labeledby attributes should be connected by ID to an element that either has text description or an "aria-label" attribute.

- [ ] drop_front
- [ ] drop_ingredients
- [ ] drop_nutrition
- [ ] drop_packaging
